### DISCUSSION TITLE
Hide new go forward/back button and search bar - VSCode web

### DISCUSSION THREAD

**Discussion by proyacerta** [community/community#62525] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I am using Codespaces web. There is a large zone at the top with go back and forward buttons, a search bar, and layout buttons. I do not remember it being present.

Can this area be turned off? That is completely hidden.
![Screenshot 2023-08-02 at 4 31 02 PM](https://github.com/community/community/assets/132306648/345a5acf-af65-4cfb-949b-2a0ab02e5159)

**Comment by patroytall** [comment]
Same topic on Reddit: https://www.reddit.com/r/vscode/comments/whuoms/how_can_i_addenable_the_search_bar_in_the_title/

**Comment by proyacerta** [comment, marked as answer]
In the settings, search for "Command Center". You can then disable it.
